README.md

```diff
@@ -531,11 +531,17 @@ vector.obj(pt=1, phi=1.3, eta=2, mass=5).deltaR(
 )
 ```
 
-The opposite—using low-dimensional vectors in operations defined for higher numbers of dimensions—is sometimes defined. In these cases, a zero longitudinal or temporal component has to be imputed.
+For a few operations - `+`, `-`, `==`, `!=`, ... - the dimension of the vectors should be equal. This can be achieved by using the `like` method, `to_{coordinate_name}` methods, `to_Vector*D` methods. The `to_Vector*D` methods provide more flexibility to the users, that is, new coordinate values can be passed into the methods as named arguments.
 
 ```python
-vector.obj(x=1, y=2, z=3) - vector.obj(x=1, y=2)
-vector.obj(x=1, y=2, z=0).is_parallel(vector.obj(x=1, y=2))
+v1 = vector.obj(x=1, y=2, z=3)
+v2 = vector.obj(x=1, y=2)
+
+v1 - v2.like(v1), v1.like(
+    v2
+) - v2, v1 - v2.to_xyz(), v1.to_xy() - v2, v1 - v2.to_Vector3D(
+    z=3
+), v1.to_Vector2D() - v2
```

Comment:
This might be required for black-compliant formatting, but it makes what would be easy to read into something difficult to read.

Do you need the commas?

```suggestion
v1 - v2.like(v1)
v1.like(v2) - v2
v1 - v2.to_xyz()
v1.to_xy() - v2
v1 - v2.to_Vector3D(z=3)
v1.to_Vector2D() - v2
```

Should there be some explanatory text between these?